axolotl-web/src/store/store.js

```diff
@@ -133,6 +135,9 @@ export default new Vuex.Store({
         this.commit("SET_REGISTRATION_STATUS", "pin");
         router.push("/pin")
         state.requestPin = true;
+      } else if (type == "getCaptchaToken") {
+        this.commit("SET_REGISTRATION_STATUS", "verificationCode");
```

Comment:
`verificationCode` already existed, is it because you started to work on captcha before but never finisher, or are you using it for two things now?

Reply:
Same reason as comment before. Maybe it's not needed anymore.